content/en/Community/_index.md

```diff
@@ -50,4 +50,118 @@ Kyverno is currently a CNCF sandbox project is growing fast. To qualify for the
 
 The goal of this adopters program is to gather real-world usage examples and help Kyverno achieve the incubating state criteria. We hope to achieve this by learning from the existing and new adopters of Kyverno who are willing to share their stories with CNCF. The adopters can remain private or choose to be listed in the public [Adopters List](https://github.com/kyverno/kyverno/blob/main/ADOPTERS.md).
 
-To participate, kindly fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSeMJxbK6eQ-MpsbaXZ3GKp8NEz9F932keL2pwznI1JIfi7Iqw/viewform) (It will take 5 mins of your time).
\ No newline at end of file
+To participate, kindly fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSeMJxbK6eQ-MpsbaXZ3GKp8NEz9F932keL2pwznI1JIfi7Iqw/viewform) (It will take 5 mins of your time).
+
+
+### Project Governance
+
+This document identifies the role and responsibilities for the Kyverno community members. It also highlights the requirements for anyone who is looking to take on leadership roles in the Kyverno project.
+
+In order to perform any actions on the Kyverno Github repositories, contributors should have certain access to GitHub, such as creating a pull request in a repository or changing an organization's billing settings, a contributor must have sufficient access to the relevant account or resource.
+
+**Note:** Please make sure to read and observe our [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+### Leadership Roles
```

Comment:
How are these roles mapped to the GitHub permissions? Do we need to add custom roles?

These are the current permissions in kyverno/kyverno repo.

>Read
Can read and clone this repository. Can also open and comment on issues and pull requests.

>Triage
Can read and clone this repository. Can also manage issues and pull requests.

>Write
Can read, clone, and push to this repository. Can also manage issues and pull requests.

>Maintain
Can read, clone, and push to this repository. They can also manage issues, pull requests, and some repository settings.

>Admin
Can read, clone, and push to this repository. Can also manage issues, pull requests, and repository settings, including adding collaborators.

Reply:
I think we would need to create custom roles. 
The reason we used Approver instead of Triage is to simplify the role and responsibilities for CONTRIBUTORS.

Reply:
Please update this section and I can help add custom roles.

Reply:
Hi @Anita-ihuman - it would be great to map the project roles to GitHub roles somewhere in the text:

https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization

Would this be the correct mapping?

* Contributor -> Triage
* Approver -> Write
* Maintainer -> Maintain
* Admin -> Admin